content/en/docs/rules/style-guide.md

```diff
@@ -137,23 +137,64 @@ For the output fields, expect that each Falco release typically exposes new [sup
 
 ```yaml
-output: >
-    Read monitored file via directory traversal (user=%user.name uid=%user.uid user_loginuid=%user.loginuid 
-    process=%proc.name proc_exepath=%proc.exepath command=%proc.cmdline pid=%proc.pid parent=%proc.pname 
-    file=%fd.name fileraw=%fd.nameraw gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] 
-    terminal=%proc.tty container_id=%container.id image=%container.image.repository namespace=%k8s.ns.name pod_name=%k8s.pod.name)
+user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags
 ```
 
-When writing a rule for container workloads, you should include the fields we automatically fetch from the container runtime:
+For network-related rules include:
 
 ```yaml
-output: container_id=%container.id image=%container.image.repository namespace=%k8s.ns.name pod_name=%k8s.pod.name
+connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type
 ```
 
-And as a general guidance, fields like `%evt.type`, `%proc.name`, `%proc.tty`, `%proc.cmdline` and fields related to process lineage such as `proc.aexepath[2]`, or fields around user information `%user.uid`, `%user.loginuid` are often of generic relevance to many rules. For specialized use cases, generic fields such as `%container.ip` or `%container.cni.json` can be further helpful for incident response, especially concerning non-network syscall related alerts in Kubernetes. These fields can be correlated, for example, with network proxy logs. 
+For file-related rules include:
+
+```yaml
+file=%fd.name
+```
+... and additional specialized fields from the raw args if applicable, such as `newpath=%evt.arg.newpath` for non-file descriptor events like symlinking or renaming.
+
+<!--
+TODO: Edit after Falco 0.36.0 release:
+
+For file-related rules include:
+
+```yaml
+file=%fd.name
+```
+
+Alternatively, you can explore more recent `fs.path.*` variants to simplify the consistent logging of file or directory paths, even for non-file descriptor events like symlinking or renaming. Previously, tapping into the raw args with `newpath=%evt.arg.newpath` was required for such events.
+-->
+
+When writing a rule for container workloads, you should include the fields we automatically fetch from the container runtime (`-k/-K` is *not* required here). All of these fields are incredibly crucial for effective incident response and play a vital role in determining the workload owner:
```

Comment:
```suggestion
When writing a rule for container workloads, you should include the fields we automatically fetch from the container runtime. Falco supports a special placeholder for this purpose: 

```yaml
%container.info
\```

This special placeholder will be automatically replaced with common container-related fields and, when used in conjunction with `-pk` (see the [CLI arguments page](https://falco.org/docs/reference/daemon/cli-arguments/) for more details). It will also add [basic Kubernetes fields](https://falco.org/docs/reference/rules/supported-fields/#field-class-k8s) (this does *not* require the *Kubernetes Metadata Enrichment*, i.e. `-k/-K`,  functionality to be enabled). Using `%container.info` in conjunction with `-pk` is equivalent to:
```

Note I'd to put `\` at line 175 to avoid escaping issue with the GH UI. Please, remove it.